### PR TITLE
Orphanet Alignment for March release

### PR DESCRIPTION
**Pull request by sabrinatoro**
Document is: https://docs.google.com/spreadsheets/d/17fX1oh-JSDLkPxEBTuiSXpiLx45qZI-S0spLa29ZGho/edit#gid=70424824

Files used are from Feb 20, 2024

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
 xref: OMIM:620690 {source="MONDO:equivalentTo"}
 is_a: MONDO:0000158 {source="OMIM:620690"} ! developmental dysplasia of the hip
 
+[Term]
+id: MONDO:0958083
+name: conjoined twins
+xref: Orphanet:647916 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019755 {source="Orphanet:647916"} ! developmental defect during embryogenesis
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:526570]: if it is not too much trouble to re-run this, I think we started also adding our ORCID as the source, so we won't flag these later as ordo-only superclass assertions that need to be reviewed

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
...
+[Term]
+id: MONDO:0958094
+name: adult-onset progressive leukoencephalopathy-early-onset deafness
+def: "A rare genetic neurological disorder characterized by congenital or early-onset sensorineural deafness and adult-onset progressive leukoencephalopathy. Progressive cognitive impairment and behavioral abnormalities are observed in the second or third decade of life, sometimes preceded by mild developmental delay and learning difficulties. Visual impairment in adult age has been reported. No central nervous system calcification is reported." [Orphanet:652532]
+xref: Orphanet:652532 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019046 {source="Orphanet:652532"} ! leukodystrophy
+
+[Term]
+id: MONDO:0958106
+name: congenital insensitivity to pain syndrome, marsili type
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:526588]: this is something that needs to be done as part of this process - the labels need to be reviewed for capitalization, this should probably be Marsili not marsili

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
...
+[Term]
+id: MONDO:0958110
+name: atrophic papulosis
+xref: Orphanet:656071 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019293 {source="Orphanet:656071"} ! skin vascular disease
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:0958115
+name: autosomal recessive combined immunodeficiency due to complete il6st deficiency
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:526601]: this gene name needs to be capitalized appropriately

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
...
+
+[Term]
+id: MONDO:0958116
+name: autosomal recessive combined immunodeficiency due to partial il6st deficiency
+xref: Orphanet:656300 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015131 {source="https://orcid.org/0000-0002-4142-7153"} ! combined immunodeficiency
+is_a: MONDO:0018037 {source="Orphanet:656300"} ! hyper-IgE syndrome
+
+[Term]
+id: MONDO:0958117
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:526615]: fix gene name

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
...
+name: autosomal dominant combined immunodeficiency due to partial il6st deficiency
+xref: Orphanet:656313 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015131 {source="https://orcid.org/0000-0002-4142-7153"} ! combined immunodeficiency
+is_a: MONDO:0018037 {source="Orphanet:656313"} ! hyper-IgE syndrome
+
+[Term]
+id: MONDO:0958118
+name: autosomal recessive combined immunodeficiency due to il6r deficiency
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:526623]: fix gene name

File: src/ontology/mondo-edit.obo [hunk 6]
```diff
@@ -526538,6 +526563,89 @@ name: developmental dysplasia of the hip 3
...
+[Term]
+id: MONDO:0958119
+name: embryonal tumor with multilayered rosettes
+def: "A rare central nervous system embryonal tumor characterized by embryonal cells arranged in multilayered rosettes and displaying one of three morphological patterns: embryonal tumor with abundant neuropil and true rosettes, ependymoblastoma, or medulloepithelioma. The tumors typically have a C19MC alteration or (rarely) a DICER1 mutation and correspond to WHO grade IV. They are mostly localized intracranially, rarely in the spinal cord, and commonly cause signs and symptoms of elevated intracranial pressure, sometimes seizures and focal neurological signs. Most cases occur in children during the first two years of life." [Orphanet:656417]
+xref: Orphanet:656417 {source="MONDO:equivalentTo"}
+is_a: MONDO:0016713 {source="Orphanet:656417"} ! central nervous system Ewing sarcoma/peripheral primitive neuroectodermal tumor
+
+[Term]
+id: MONDO:0958120
+name: autosomal dominant combined immunodeficiency due to erbin deficiency
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:526637]: maybe Erbin needs to be capitalized?

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
some minor updates needed to labels

you may want to consider adding your ORCID as the source with the Orphanet sources (this seems less critical now though after our mass obsoletion work, but probably still a good idea/best practice)

**Comment by nicolevasilevsky** [comment]
@sabrinatoro if you don't have time to address these issues before you leave for vacation, I can take care of it

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:526588]: Thank you for catching this! I missed the capitals. :-)

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:526570]: done :-)

**Comment by sabrinatoro** [comment]
@nicolevasilevsky, changes have been made. It is ready for your review again. Thank you!

**Review by nicolevasilevsky — APPROVED** [review]
looks great, woohoo!

**Pull request merged**